pkgs/development/python-modules/pystemmer/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, python, fetchPypi, buildPythonPackage, cython }:
+
+buildPythonPackage rec {
+  pname = "PyStemmer";
+  version = "1.3.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "d1ac14eb64978c1697fcfba76e3ac7ebe24357c9428e775390f634648947cb91";
+  };
+
+  nativeBuildInputs = [ cython ];
+
+  preBuild = ''
+    cython src/Stemmer.pyx
```

Comment:
I hope so we do not add cython to some critical dependency chains though.

Reply:
That should not be the case.

Their `setup.py` doesn't automatically compile it when cython is present?

Reply:
In theory yes. I will try to remove `preBuild`: https://github.com/snowballstem/pystemmer/blob/master/setup.py#L46